### PR TITLE
feat(FX-3049): nav bar update

### PR DESCRIPTION
**Pull request by tr-ann**
Jira: [FX-3049](https://artsyproduct.atlassian.net/browse/FX-3049)

Description:
The Curatorial team is requesting an update to the Artworks and Artists Nav Bars at the top of artsy.net. We've reviewed the performance of existing categories and collections, and removed low performing content and introduced new content that has historically performed well. We're hoping that this update can take place in early Q3.

Link to requested updates: https://docs.google.com/spreadsheets/d/1EOYineDRM9QTfNE7gUyO3pnqKUrr_pgekCHYsWsxK-U/edit#gid=0

_ARTIST_
**Artist Nav Before:**
image

**Artist Nav After:**
image

_ARTWORK_
**Artwork Nav Before**
image

**Artwork Nav After**
image

### CODE REVIEW

**Review by dimatretyak — APPROVED** [review]
LGTM 👍

File: src/v2/Components/NavBar/menuData.ts [hunk 1]
```diff
@@ -25,20 +25,16 @@ export const ARTISTS_SUBMENU_DATA: MenuLinkData = {
...
       {
-        text: "Portrayals of Queer Love",
-        href: "/collection/queer-love-pride-month-2020",
+        text: "In-Demand Artists",
+        href: "/collection/in-demand-artists",
       },
       {
-        text: "Contemporary Women Surrealists",
-        href: "/collection/contemporary-women-surrealists",
+        text: "Artist's We're Eyeing",
```
Comment (mdole) [inline src/v2/Components/NavBar/menuData.ts:32]: Ah, this should actually be "Artists We're Eyeing" (it's incorrect in the spreadsheet too, but you can see on the [actual page](https://www.artsy.net/collection/artists-we-are-eyeing) how it should read)

File: src/v2/Components/NavBar/menuData.ts [hunk 2]
```diff
@@ -270,8 +270,8 @@ export const ARTWORKS_SUBMENU_DATA: MenuLinkData = {
               href: "/collection/sculpture",
             },
             {
-              text: "Drawing",
-              href: "/collection/drawing",
+              text: "Work on Paper",
```
Comment (mdole) [inline src/v2/Components/NavBar/menuData.ts:273]: Hmmm this is a tough one...I think it should also be `Works on Paper`. Let's make that change, and I'll confirm with the team responsible

**Review by mdole — CHANGES_REQUESTED** [review]
@tr-ann Thanks for picking this up! Two small changes (neither of which are on you - they were mistakes in the original spreadsheet) and we'll be ready to merge :)

**Comment by tr-ann** [comment]
@mdole thanks for the comments. I made the changes so can you review it again? :)

**Review by mdole — APPROVED** [review]
Perfect thanks! #squashongreen

**Pull request merged**